CHANGELOG.md

```diff
@@ -4,8 +4,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [0.36.0](https://github.com/AliceO2Group/Bookkeeping/releases/tag/%40aliceo2%2Fbookkeeping%400.36.0)
 * Notable changes for users:
-  * A detail page has been created for LHC fills and is accessible from LHC fills overview, run details and runs
-    overview
+  * A detail page has been created for LHC fills and is accessible from LHC fills overview, run details and runs overview
```

Comment:
Please add a few words on the statistics

Reply:
Done